client/src/components/Collections/PairedListCollectionCreator.vue

```diff
@@ -947,9 +948,9 @@ function _naiveStartingAndEndingLCS(s1: string, s2: string) {
                             {{
                                 localize(
                                     [
-                                        "Collections of paired datasets are ordered lists of dataset pairs (often forward and reverse reads). ",
-                                        "These collections can be passed to tools and workflows in order to have analyses done on each member of ",
-                                        "the entire group. This interface allows you to create a collection, choose which datasets are paired, ",
+                                        "Lists of pairs are an ordered list of individual dataset paired together in their own collection (often forward and reverse reads). ",
```

Comment:
```suggestion
                                        "List of pairs is an ordered list of individual datasets paired together in their own collection (often forward and reverse reads). ",
```

Reply:
using singular sounds clearer to me here, but feel free to disregard

Reply:
I think I could agree if it was "A list of pairs" and below "A list" (instead of "This list"). To me "List of pairs is" feels odd. I'll commit that change - it stills satisfies your point that singular would sound better - and I think agree with that.

Reply:
Ahh... reading it in the broader context I think we are describing lists generally in that first sentence. What if we kept the plural here but we added a sentence at the beginning like "This interface allows you to build a new Galaxy list of pairs. List of pairs are an ordered....". If that still reads odd to you - I'll try something else - I really appreciate your perspective here and for you taking the time to comment. I want this language to read well and feel universal. The language around collections has repeatedly escaped me in ways I'm uncomfortable with.

Reply:
I like the added sentence. It is hard (and important imho) to distinguish where we are describing a concrete interface user is looking at and where we explain broader concepts.